nvgpu/__init__.py

```diff
@@ -5,7 +5,7 @@
 
 def gpu_info():
     gpus = [line for line in _run_cmd(['nvidia-smi', '-L']) if line]
-    gpu_infos = [re.match('GPU ([0-9]+): ([^(]+) \(UUID: ([^)]+)\)', gpu) for gpu in gpus]
+    gpu_infos = [re.match('GPU ([0-9]+): ([^.]+) \(UUID: ([^)]+)\)', gpu) for gpu in gpus]
```

Comment:
It could be changed in more a robust way to: https://github.com/rossumai/nvgpu/issues/24#issuecomment-1259532626.

Reply:
i apply user advice https://github.com/rossumai/nvgpu/issues/24#issuecomment-1259532626.